### ISSUE TITLE
[Bug]: migrations missing in v1.44.28 containers for litellm proxy when migrating from v1.41.24

### ISSUE THREAD

**Issue by TensorTemplar** [BerriAI/litellm#5677]
### What happened?

How to reproduce:
1. run `v1.44.28` container with external db and config option
```yaml
    general_settings:
      store_model_in_db: True
```

expected:
container starts, migrates automatically

actual:
container fails to start (see logs below)

tested this on k8s migrating from version 1.41.24, maybe somewhere in between the migration was applied. It may be worth pointing out somewhere what upgrade paths are supported, but not critical.

I also tested with a clean db but the error there just changes to no migrations found at all.

Workaround:
1. copy [create_views.py](https://github.com/BerriAI/litellm/blob/main/db_scripts/create_views.py) into the container
2. remove the [db env line](https://github.com/BerriAI/litellm/blob/cdd7cd4d69585525bc9aaa0e647cdbb38b3ae6c0/db_scripts/create_views.py#L9C13-L9C26) - we are all adults here so presumably `DATABASE_URL` is passed via env
3. run `python create_views.py`
startup works after this

### Relevant log output

```shell
INFO:     Started server process [1]
INFO:     Waiting for application startup.
ERROR:    Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 732, in lifespan
    async with self.lifespan_context(app) as maybe_state:
  File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 608, in __aenter__
    await self._router.startup()
  File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 709, in startup
    await handler()
  File "/usr/local/lib/python3.11/site-packages/litellm/proxy/proxy_server.py", line 2918, in startup_event
    create_view_response = await prisma_client.check_view_exists()
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/backoff/_async.py", line 151, in retry
    ret = await target(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/litellm/proxy/utils.py", line 992, in check_view_exists
    raise e
  File "/usr/local/lib/python3.11/site-packages/litellm/proxy/utils.py", line 986, in check_view_exists
    raise Exception(
Exception: Not all views exist in db. Got=6. Expected=7. Run 'create_views.py' in litellm/db_scripts to create missing views.

ERROR:    Application startup failed. Exiting.
```

**Comment by ghost** [comment]
Hi @TensorTemplar this is fixed in v`1.45.0` 

We handle the required view creation, but emit a warning for missing optional views (reason: trying to create missing views on large db's caused proxy startup errors).

**Comment by ishaan-jaff** [comment]
Hi @TensorTemplar , curious do you use LiteLLM today ? If so, I'd love to hop on a call and learn how we can improve LiteLLM for you

- my cal for your convenience: https://calendly.com/d/4mp-gd3-k5k/litellm-1-1-onboarding-chat
- my linkedin if you prefer DMs: https://www.linkedin.com/in/reffajnaahsi/
